docs/user-guide.md

```diff
@@ -1,16 +1,18 @@
 # User guide
 
+- [FAQ](/docs/user-guide/faq.md) - frequently asked questions about using and configurating stylelint.
```

Comment:
I'd put configuring first, `configuring and using`, just cuz there's some chronological sense to that.